app/services/reports/region_service.rb

```diff
@@ -42,8 +42,8 @@ def call
       result.uncontrolled_patients_with_ltfu_rate = repository.uncontrolled_rates(with_ltfu: true)[slug]
 
       result.visited_without_bp_taken = repository.visited_without_bp_taken[region.slug]
-      result.calculate_percentages(:visited_without_bp_taken)
-      result.calculate_percentages(:visited_without_bp_taken, with_ltfu: true)
+      result.visited_without_bp_taken_rate = repository.visited_without_bp_taken_rate[region.slug]
+      result.visited_without_bp_taken_with_ltfu_rate = repository.visited_without_bp_taken_rate(with_ltfu: true)[region.slug]
```

Comment:
This is the (hopeful) bug fix - we no longer need to recalc the percentage, as repository already has done that for us.